### PR TITLE
Update to test_track_js_client 1.3.6

### PR DESCRIPTION
**Pull request by chrislopresto**

### CODE REVIEW

**Comment by jmileham** [comment]
LGTM! Worth cutting a version of the rails client or should we wait and roll up some more change?

File: bower.json [hunk 1]
```diff
@@ -12,6 +12,6 @@
     "tests"
   ],
   "dependencies": {
-      "test_track_js_client": "https://github.com/Betterment/test_track_js_client.git#v1.3.3"
+      "test_track_js_client": "https://github.com/Betterment/test_track_js_client.git#v1.3.6"
```
Comment (samandmoore) [inline bower.json]: We can swap this to pull from bower, right? Or, we still have to publish the js client to bower. I will add that as an issue on the js repo.

**Comment by aburgel** [comment]
bump on this. we also need to run `rake vendor_deps` and commit the minified js client

**Comment by aburgel** [comment]
#13 upgraded the js client to 1.3.7. So this is obsolete.

File: bower.json (hunk 1 above)
Comment (chrislopresto) [inline bower.json]: I'd say yes. Let's publish to bower as part of our release process.

**Pull request closed without merging**